charts/auth-layer-proxy/values.yaml

```diff
@@ -0,0 +1,47 @@
+affinity: {}
+
+command: ["/etc/envoy/start-envoy.sh"]
+
+configEnv:
+  CLIENT_ID: "htg-auth-layer"
+  CLIENT_SECRET: "2IfFX7eqQvg2wY2hh1qjtS8RrUY9YqEg"
```

Comment:
nit: there should be a comment to highlight that this a configured default or a random value that should be changed per deployment. I think you may have covered it in documentation so you can also put a link to that section

Reply:
It does not make sense to provide a random value, since this needs to come from the auth-server, that said I did some changes so it can be provided as a local value or a global value (in preparation for sharing with other charts, in this case the server).

Also added documentation explaining the change and a validation that will fail in case neither of the values is provided, with a descriptive message.

```
A valid client secret must be provided either via .Values.global.auth.clientSecret or .Values.configSecrets.clientSecret

```